src/Ledger/EssentialAgda.lagda

```diff
@@ -2,7 +2,8 @@
 \label{sec:appendix-agda-essentials}
 Here we describe some of the essential concepts and syntax of the Agda programming language and proof assistant.
 The goal is to provide some background for readers who are not already familiar with Agda, to help them
-understand the other sections of the specification.
+understand the other sections of the specification.  For more details, the reader is
+encouraged to consult the official Agda documentation~(\cite{agda2024}).
```

Comment:
I feel this should include a link to the documentation besides the citation.

Reply:
I've taken your suggestion and created a macro similar to the one you suggest below, but the one I propose is flexible enough to use for both general reference to the Agda docs, and to a specific subsection of the docs.  Here it is:

```tex
\NewDocumentCommand{\hrefAgdaDocs}{O{} O{Agda documentation}}{%
  \href{https://agda.readthedocs.io/en/\AgdaVersion/#1}{#2}%
}
```

So we can use `\hrefAgdaDocs{}` to link to the main Agda docs page and `\hrefAgdaDocs[URLsuffix]{Link text}` to link to a specific section of the Agda docs.

For example, `\hrefAgdaDocs[language/record-types.html][agda.readthedocs.io/record-types]` will produce the link [agda.readthedocs.io/record-types](https://agda.readthedocs.io/en/2.7.0/language/record-types.html) in the pdf.